core/src/main/scala/com.softwaremill.kmq/KafkaClientsResourceHelpers.scala

```diff
@@ -0,0 +1,37 @@
+package com.softwaremill.kmq
+
+import cats.effect.{IO, Resource}
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.common.serialization.{Deserializer, Serializer}
+
+import scala.jdk.CollectionConverters._
+
+final class KafkaClientsResourceHelpers(val clients: KafkaClients) {
```

Comment:
this is used only from `redelivery`, let's move it there and make private (as it's a helper)